zp-relayer/config.ts

```diff
@@ -48,6 +48,7 @@ const config = {
     .map(s => parseInt(s, 10)),
   requireTraceId: process.env.RELAYER_REQUIRE_TRACE_ID === 'true',
   requireHTTPS: process.env.RELAYER_REQUIRE_HTTPS === 'true',
+  logIgnoreRoutes: (process.env.RELAYER_LOG_IGNORE_ROUTES || '').split(' ').filter(r => r.length > 0),
```

Comment:
Consider to make separators consistent for all configuration options. The space is the separator in the lists `RPC_URL` and `RELAYER_LOG_IGNORE_ROUTES`. Whereas the comma is the separator in `RELAYER_JSONRPC_ERROR_CODES`.

Reply:
Changed separator to space